snapcraft.yaml

```diff
@@ -23,7 +24,7 @@ parts:
       ./setup.sh hints/linux
       cd ../../
       snapcraftctl build
-      cp ../../../nethack.sh $SNAPCRAFT_PART_INSTALL/
+      cp ../../../project/nethack.sh $SNAPCRAFT_PART_INSTALL/
```

Comment:
this won't work (there is no dir /project in the toplevel of the tree) ... could you change it to:
```
-      cp ../../../nethack.sh $SNAPCRAFT_PART_INSTALL/
+      cp $SNAPCRAFT_PROJECT_DIR/nethack.sh $SNAPCRAFT_PART_INSTALL/
```

Reply:
I can, but it won't build on my system without that change.

Latest snapcraft snap on 20.04 here, using multipass integration.

Does it work for you without?